compiler/rustc_parse/src/parser/mod.rs

```diff
@@ -321,16 +321,16 @@ enum TokenType {
 }
 
 impl TokenType {
-    fn to_string(&self) -> String {
+    fn to_string(&self) -> std::borrow::Cow<'static, str> {
```

Comment:
Maybe rename to `to_cow`?

Reply:
`to_cow` is significantly less descriptive, because the fact that it's a `Cow` is the least important component of the type, and `to_cow_str` is way too verbose. I think we should keep `to_string`.